README.md

```diff
@@ -5,10 +5,13 @@ TL;DR: AzDO Pipeline Template wrapping terraform apply and plan with a manual ve
 > [!WARNING]
 > The terraform plan cannot foresee every possible circumstance and scenario when determining changes to the infrastructure, only the terraform apply will truly know what changes will be made. Always test in an development environment first before moving to production.
 
+> [!IMPORTANT]
```

Comment:
Would it be sensible to set the default container to this as well?

Reply:
I wondering whether I should make the container an optional thing? Should a template prescribe the pool/container at all or be designed to be flexible as possible? How out of the box to make it 🤔

Reply:
sorry hit resolved by mistake, I think if we say is should be using this container, and that container is public if this ever goes or is public the default still stands, setting the default does not then mean no one has an option to select a different container.

Reply:
its just a case of setting what we say in the readme in here:

  - name: TerraformContainer
    type: string
    default: ''